code/modules/busy_space/systems.dm

```diff
@@ -0,0 +1,602 @@
+		)
+
+/datum/lore/system/abels_rest
+	name = "Abel's Rest"
+	desc = "The Hegemony invaded and occupied this system in 2508 and somehow has resisted all human efforts to detail it further"
+	locations = list(
+		new /datum/lore/location("a Fleet base in Abel's Rest", list(ATC_TRANS, ATC_INDU, ATC_FREIGHT)),
+		new /datum/lore/location("an Unathi exclave in Abel's Rest", list(ATC_TYPICAL, ATC_DIPLO)),
+		new /datum/lore/location("a First Contact derelict in Abel's Rest", list(ATC_SALVAGE)),
+		)
+
+/datum/lore/system/zhu_que
+	name = "Zhu Que"
+	desc = "It's in the Inner Bowl and, likewise, has resisted all efforts to detail it further."
+	planets = list("Jade")
+	locations = list(
+		new /datum/lore/location("Urgia in Zhu Que", list(ATC_ALL_CIV)), //ancient notes
+		new /datum/lore/location("a Ward-Takahashi plant in Zhu Que", list(ATC_TYPICAL)),
+		)
+
+/datum/lore/system/zhu_que/New()
+	planetary_destinations -= "a dome" //Jade is fully habitable
+	..()
+
+//Almach Protectorate
+/datum/lore/system/relan
+	name = "Relan"
+	desc = "The first colonized system in the Almach Rim and defacto leader of the floudering Protectorate. Politically divided between Taron, the system's main planet, and Relan, its spacebound population"
+	planets = list("Taron")
+	locations = list(
+		new /datum/lore/location("Maruya in Relan", list(ATC_INDU)),
+		new /datum/lore/location("New Dayton on Taron, Relan", list(ATC_ALL_CIV)),
+		new /datum/lore/location("New Garissa on Taron, Relan", list(ATC_TYPICAL)),
+		new /datum/lore/location("Perez Rest on Taron, Relan", list(ATC_TYPICAL)),
+		new /datum/lore/location("Parker in Relan", list(ATC_TYPICAL)),
+		new /datum/lore/location("Abhayaranya in Relan", list(ATC_TYPICAL)),
+		new /datum/lore/location("Carter Interstellar Spaceport, Relan", list(ATC_ALL_CIV, ATC_DIPLO)),
+		new /datum/lore/location("a pirate base in Relan", list(ATC_DEF)),
+		new /datum/lore/location("a Protectorate laboratory in Relan", list(ATC_SCI)),
+		new /datum/lore/location("a Kaleidoscope Cosmetics office in Relan", list(ATC_TRANS, ATC_SCI)),
+		new /datum/lore/location("an Almach War derelict in Relan", list(ATC_SALVAGE)),
+		)
+
+/datum/lore/system/relan/New()
+	locations += list(
+		new /datum/lore/location("[pick("Fu Xi", "Nuwa", "Shennong")], Xia subsystem, Relan", list(ATC_TYPICAL)), //identically undetailed sattelites get compressed
+		new /datum/lore/location("[pick("Ubaid", "Badari", "Harappan", "Olmec", "Caral")], Zegev subsystem, Relan", list(ATC_TYPICAL)),
+		)
+	..()
+
+/datum/lore/system/vounna
+	name = "Vounna"
+	desc = "Home system of the Prometheans and member of the Protectorate."
+	planets = list("Aetolus")
+	locations = list(
+		new /datum/lore/location("the Spear of Agrafa in Vounna", list(ATC_TYPICAL)),
+		new /datum/lore/location("the Prometheus in Vounna", list(ATC_TYPICAL, ATC_DIPLO)),
+		new /datum/lore/location("the remains of the SGWP Delilah in Vounna", list(ATC_SALVAGE)),
```

Comment:
What is this in reference to? It seems a little hyper-specific of a wreck if it isn't something large and significant.

Reply:
the SGWP Delilah was the sol gov base in Vounna, disabled during the coup